src/v6/signing.ts

```diff
@@ -0,0 +1,253 @@
+import { _TypedDataEncoder } from "ethers/lib/utils"
+import { keccak_256 } from "@noble/hashes/sha3"
+import { hexToBytes, utf8ToBytes } from "@noble/hashes/utils"
+import { AllowFill, GenericSwapData, LimitOrder, RFQOffer, RFQTx } from "./types"
+
+import {
+    EIP712DomainOptions,
+    EIP712Types,
+    SigningHelper as BaseSigningHelper,
+    SigningOptions,
+} from "../signing"
+
+export class SigningHelper extends BaseSigningHelper {
+    public constructor() {
+        super({
+            name: "Tokenlon",
+            version: "v6",
+        })
+    }
+
+    private toBuffer = (data: string, encoding: "utf8" | "hex" = "hex"): Uint8Array => {
```

Comment:
Would be better to stick to same function declaration pattern

```suggestion
    private toBuffer(data: string, encoding: "utf8" | "hex" = "hex"): Uint8Array {
```

Reply:
Fixed!